[RHCLOUD-35780] Make Drawer notifications actionable by adding a link to additional details (#4405)
RedHatInsights/notifications-backend: common-template/src/main/resources/templates/drawer/Rbac/roleRemovedFromPlatformGroupBody.md

> Red Hat removed a role **[{data.events[0].payload.role.name}]({environment.url}/iam/user-access/roles/detail/{data.events[0].payload.role.uuid}?from=notifications&integration=drawer)** from platform default access group.

Reviewer: ```suggestion
Red Hat removed the **[{data.events[0].payload.role.name}]({environment.url}/iam/user-access/roles/detail/{data.events[0].payload.role.uuid}?from=notifications&integration=drawer)** role from the Default access group.
```
